src/components/Amplitude.js

```diff
@@ -1,149 +1,43 @@
-import debounce from 'lodash.debounce';
-import PropTypes from 'prop-types';
 import React from 'react';
```

Comment:
I'm nitpicking here, but generally more correct form (& future proof) would be `import * as React from 'react'` (same for PropTypes), currently both works due to CJS / ESM compatibility drama, but this might stop working in the future